build.gradle

```diff
@@ -6,6 +6,7 @@ plugins {
 
 allprojects {
     apply plugin: 'idea'
+    apply plugin: 'eclipse'
```

Comment:
The eclipse plugin is deprecated for removal. Why are we adding it in at this juncture?

Reply:
Good catch. gone

Reply:
Saxon always sneaks this in because he is an eclipse user.